docs/source/guide/custom_metric.md

```diff
@@ -105,4 +107,137 @@ After adding your code to Label Studio Enterprise, the following could happen:
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+                "lambda:CreateFunction",
+                "lambda:UpdateFunctionCode",
+                "lambda:InvokeFunction",
+                "lambda:GetFunction",
+                "lambda:DeleteFunction"
+            ],
+            "Resource": [
+      "arn:aws:lambda:*:YOUR_AWS_ACCOUNT:function:custom-metric-*"
+            ]
+        },
+        {
+            "Sid": "VisualEditor2",
+            "Effect": "Allow",
+            "Action": "lambda:ListFunctions",
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+How you attach the policy to the proper `label-studio` role depends on how you deployed Label Studio Enterprise in your self-managed cloud infrastructure. 
+- [Deployed with Docker Compose running in EC2](#Deployed-with-Docker-Compose-running-in-EC2)
+- [Deployed in EKS with an OIDC provider](#Deployed-in-EKS-with-an-OIDC-provider)
+- [Deployed in EKS without an OIDC provider](#Deployed-in-EKS-without-an-OIDC-provider)
+
+### Deployed with Docker Compose running in EC2
+
+If you deployed Label Studio Enterprise using Docker Compose in an AWS EC2 instance, do the following to finish setting up permissions for the custom agreement metric functionality:
+1. Create an IAM user with an Access key with Programmatic access. Save the user credentials somewhere secure.
+2. Attach the policy created in the previous step, `LSE_AllowInteractLambda` to the IAM user that you create.
+3. In the `docker-compose.yaml` file that you use to deploy Label Studio Enterprise, add the following environment variables in the `app` and `rqworkers` sections:
+> Update `YOUR_AWS_ACCESS_KEY_ID`, `YOUR_AWS_SECRET_ACCESS_KEY` and `YOUR_AWS_ACCOUNT` with the credentials for the account created in step one, and updating `YOUR_AWS_REGION` with the AWS region that your EC2 instance exists in:
+```
+AWS_ACCESS_KEY_ID=YOUR_AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY=YOUR_AWS_SECRET_ACCESS_KEY
+LS_LAMBDA_REGION_CUSTOM_METRICS=YOUR_AWS_REGION
+LS_LAMBDA_ROLE_CUSTOM_METRICS=arn:aws:iam::YOUR_AWS_ACCOUNT:role/LSE_CustomMetricsExecuteRole
+```
+
+### Deployed in EKS with an OIDC provider
+
+If you deployed Label Studio Enterprise in Amazon Elastic Kubernetes Service (EKS) with OpenID Connect (OIDC) for identity and access management (IAM), do the following to finish setting up permissions for the custom agreement metric functionality:
+1. Create an AWS IAM policy named `LSE_ServiceAccountApp`.
+2. Attach the `LSE_AllowInteractLambda` to the `LSE_ServiceAccountApp` policy. 
+3. Update your helm `values.yaml` file to include the following map. 
+> Replace `YOUR_AWS_ACCOUNT` with your AWS account ID:
+```yaml
+app:
+  serviceAccount:
+    annotations: 
+      eks.amazonaws.com/role-arn: arn:aws:iam::YOUR_AWS_ACCOUNT:role/LSE_ServiceAccountApp
+```
+4. Restart your Helm release.
```

Comment:
how?

Reply:
Good catch here!

- Run `helm list` to identify the <RELEASE_NAME> of LSE deployment
- Run these commands sequentially:
```
kubectl rollout restart deployment/<RELEASE_NAME>-lse-rqworker
kubectl rollout restart deployment/<RELEASE_NAME>-lse-app
```